charts/dapr/crds/subscription.yaml

```diff
@@ -129,6 +141,18 @@ spec:
               deadLetterTopic:
                 description: The optional dead letter queue for this topic to send events to.
                 type: string
+              bulkSubscribe:
+                description: Represents bulk subscribe properies
+                properties:
+                  enabled: 
+                    type: string
```

Comment:
Here too, I think the types should be more explicit: bool, int, int?

Reply:
Updated